.github/workflows/main.yml

```diff
@@ -38,7 +38,20 @@ jobs:
           poetry-version: ${{ matrix.poetry-version }}
       - name: Install dependencies
         run: poetry install
-      - name: Lint with flake8
+      - name: Lint and style check with flake8 and black
         run: poetry run flake8 && poetry run black --check .
       - name: Run unit tests
         run: poetry run pytest
+  integration-tests:
+    # this workflow runs tests on the full docker stack
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build docker stack
+        run: docker-compose up -d
+      - name: Check running containers
+        run: docker ps
+      - name: Run tests
+        run: docker exec airflow_sheduler pytest -v
```

Comment:
I am unsure in which container (which airflow process) we should run integration tests. Do you have experience with this @davhin ?